packages/dart/build.sh

```diff
@@ -36,6 +35,9 @@ termux_step_pre_configure() {
 	sed -i -e 's:\([^A-Za-z0-9_]\)/usr/bin:\1'$TERMUX_PREFIX'/local/bin:g' \
 		-e 's:\([^A-Za-z0-9_]\)/bin:\1'$TERMUX_PREFIX'/bin:g' \
 		"$TERMUX_PKG_SRCDIR/sdk/third_party/pkg/pub/lib/src/io.dart"
+
+        # Build dart-sdk need more disk space, delete unused files
+        rm -rf $ANDROID_HOME $NDK
```

Comment:
Don't do this, [add dart here instead](https://github.com/termux/termux-packages/blob/ca2e7bcf7361099054740f954be0d4b0b9874a06/.github/workflows/packages.yml#L153).

Reply:
this `rm` had to do, or the build will fail because disk size is no enough, and dart build system does not require these two options

Reply:
> Don't do this, [add dart here instead](https://github.com/termux/termux-packages/blob/ca2e7bcf7361099054740f954be0d4b0b9874a06/.github/workflows/packages.yml#L153).

Does this command affect the construction of other packages?

Reply:
I think so, if other packages are built after this.